styles/_base.scss

```diff
@@ -165,6 +164,7 @@ ul, ol {
         margin: 0 calc((100% - #{$pageWidth + $padLg + $padLg}) / 2) 0 auto;
         padding: 0 $padLg $bar $padLg;
         position: relative;
+        min-height: 1000px;
```

Comment:
What's the purpose of the min-height? The pagecontainer element always sizes itself to the height of the content in it.